### DISCUSSION TITLE
Identical Buff Names in the APL

### DISCUSSION THREAD

**Discussion by flollef** [simulationcraft/simc#6534] [Q&A]
Is there a way to distinguish between buffs with identical names in the APL? The example I am thinking of is the Languishing Soul Detritus buff which has two forms with identical names.

**Comment by Dorovon** [comment]
It depends on how the buff is implemented. If something is implemented as two buffs, they will have different names in simc for you to reference.

In this case, the same buff is used for both parts with a different value applied. You can check the value in the APL with `buff.languishing_soul_detritus.value`.
